packages/bootstrap-4/src/RangeWidget/RangeWidget.tsx

```diff
@@ -1,12 +1,12 @@
-import { FormContextType, getTemplate, RJSFSchema, StrictRJSFSchema, WidgetProps } from '@rjsf/utils';
+import { FormContextType, getTemplate, labelValue, RJSFSchema, StrictRJSFSchema, WidgetProps } from '@rjsf/utils';
 
 export default function RangeWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>(
   props: WidgetProps<T, S, F>
 ) {
-  const { value, label, displayLabel = true, options, registry } = props;
+  const { value, label, hideLabel, options, registry } = props;
   const BaseInputTemplate = getTemplate<'BaseInputTemplate', T, S, F>('BaseInputTemplate', registry, options);
   return (
-    <BaseInputTemplate {...props} extraProps={displayLabel ? label : undefined}>
+    <BaseInputTemplate {...props} extraProps={{ label: labelValue(label || undefined, hideLabel) }}>
```

Comment:
There was a bug here around properly passing the `label` prop. Also, passing undefined when label was falsey was better